content/en/docs/Installation/_index.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
Could we use the existing `Getting started` section for this instead of creating a new section? 

https://github.com/anik120/Operator-Lifecycle-Manager-doc-website/blob/master/content/en/docs/Getting%20started/_index.md#installation

It'd be great if you could rearrage the doc a bit too, by sticking to the arrangements of the sections there ^ 

For example, the prerequisite section could be moved over to  https://github.com/anik120/Operator-Lifecycle-Manager-doc-website/blob/master/content/en/docs/Getting%20started/_index.md#prerequisites